src/components/Button/Button.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus, faCheck, faSortDown } from '@fortawesome/free-solid-svg-icons';
+
+/* props 중 bgc와 color는 Button 컴포넌트를 사용하는 부모컴포넌트에서 theme을 import하시면 됩니다.
+ex) bgc={theme.primary} color={theme.white} 
+theme은 이미 index.js에서 import 되었지만 컴포넌트 안에서 theme을 사용하기위해 다시 import 했습니다. */
+
+function Button({
```

Comment:
`export`도 한번에 처리해주는 것은 어떨까요?

```suggestion
export default function Button({
```

Reply:
다 같이 통일한다면 저는 찬성입니다!